storage/simulation/cluster.go

```diff
@@ -272,21 +296,24 @@ func (c *Cluster) String() string {
 	return buf.String()
 }
 
-// StringEpochHeader creates the string header for epoch outputs based on all
-// of the current stores.
-func (c *Cluster) StringEpochHeader() string {
-	var buf bytes.Buffer
-	buf.WriteString("Store:\t")
+// OutputEpochHeader outputs to the epoch writer the header for epoch outputs
+// based on all of the current stores.
+func (c *Cluster) OutputEpochHeader() {
+	w := new(tabwriter.Writer)
+	w.Init(c.epochWriter, 0, 8, 0, '\t', 0)
+	fmt.Fprintf(w, "Store:\t")
 	for _, storeID := range c.storeIDs {
-		fmt.Fprintf(&buf, "%d\t", storeID)
+		fmt.Fprintf(w, "%d\t", storeID)
 	}
-	return buf.String()
+	_ = w.Flush()
```

Comment:
My plan is to output using a multiwriter to both a file and to the the console.  Flus will ensure that the buffer is clear.  If we are simulating a large cluster, it might have a lot of output, if that's the case, we want to ensure the full line is outputted before proceeding.